index.html

```diff
@@ -2261,6 +2348,19 @@ <h3>The <dfn>NDEFWriteOptions</dfn> dictionary</h3>
     </p>
   </section>
 
+  <section data-dfn-for="NDEFMakeReadOnlyOptions">
+    <h3>The <dfn>NDEFMakeReadOnlyOptions</dfn> dictionary</h3>
+    <pre class="idl">
+      dictionary NDEFMakeReadOnlyOptions {
+        AbortSignal? signal;
+      };
```

Comment:
Just a note to self. Later we might want to consider adding a safety-option here, e.g. a serial number, making sure only the given tag is made read-only. It should be taken care by the recommended practice of doing writes and making read-only from a read operation.